examples/run_glue_tpu.py

```diff
@@ -0,0 +1,490 @@
+                                        config=config,
+                                        cache_dir=args.cache_dir if args.cache_dir else None)
+
+    # Send model to TPU/XLA device.
+    model.to(args.device)
+
+    logger.info("Training/evaluation parameters %s", args)
+
+    # Training
+    if args.do_train:
+        train_dataset = load_and_cache_examples(args, args.task_name, tokenizer, evaluate=False)
+        global_step, tr_loss = train(args, train_dataset, model, tokenizer, disable_logging=disable_logging)
+        logger.info(" global_step = %s, average loss = %s", global_step, tr_loss)
+
+
+    # Saving best-practices: if you use defaults names for the model, you can reload it using from_pretrained()
+    output_dir = os.path.join(args.output_dir, 'final-xla{}'.format(xm.get_ordinal()))
+    if args.do_train:
+        # Create output directory if needed
+        if not os.path.exists(output_dir):
+            os.makedirs(output_dir)
+
+        logger.info("Saving model checkpoint to %s", output_dir)
+        # Save a trained model, configuration and tokenizer using `save_pretrained()`.
+        # They can then be reloaded using `from_pretrained()`
+        model_to_save = model.module if hasattr(model, 'module') else model  # Take care of distributed/parallel training
+        model_to_save.save_pretrained(output_dir, xla_device=True)
+        tokenizer.save_pretrained(output_dir)
+
+        # Good practice: save your training arguments together with the trained.
+        torch.save(args, os.path.join(output_dir, 'training_args.bin'))
+
+        # Load a trained model and vocabulary that you have fine-tuned
+        model = model_class.from_pretrained(output_dir)
+        tokenizer = tokenizer_class.from_pretrained(output_dir)
+        model.to(args.device)
+
+
+    # Evaluation
+    results = {}
+    if args.do_eval:
+        tokenizer = tokenizer_class.from_pretrained(output_dir, do_lower_case=args.do_lower_case)
+        checkpoints = [output_dir]
+        if args.eval_all_checkpoints:
+            checkpoints = list(os.path.dirname(c) for c in sorted(glob.glob(output_dir + '/**/' + WEIGHTS_NAME, recursive=True)))
+            logging.getLogger("transformers.modeling_utils").setLevel(logging.WARN)  # Reduce logging
+        logger.info("Evaluate the following checkpoints: %s", checkpoints)
+        for checkpoint in checkpoints:
+            global_step = checkpoint.split('-')[-1] if len(checkpoints) > 1 else ""
+            prefix = checkpoint.split('/')[-1] if checkpoint.find('checkpoint') != -1 else ""
+            
+            model = model_class.from_pretrained(checkpoint)
+            model.to(args.device)
+            result = evaluate(args, model, tokenizer, prefix=prefix, disable_logging=disable_logging)
+            result = dict((k + '_{}'.format(global_step), v) for k, v in result.items())
+            results.update(result)
+
+    return results
+
+
+def get_args():
```

Comment:
Not sure what you mean by "import that function". The way they had it in the other GLUE task runner looks like this: https://github.com/huggingface/transformers/blob/master/examples/run_glue.py#L334-L418. All other flags here will be used by this TPU runner script.

Reply:
Ah I see. If they had `get_args` function as you have here, you could import that function and use it here to avoid duplication. You can maybe upstream that change later. for now it's fine.

Reply:
Ack.